src/content/1.7/modules/creation/module-translation/new-system.md

```diff
@@ -140,11 +140,11 @@ Since the module is called MyModule, the translation domain should be `Modules.M
 ```php
 // file: controllers/front/something.php
 
-class MyModuleSomethingFrontController extends ModuleFrontController
+class MySomethingModuleFrontController extends ModuleFrontController
 {
     public function initContent()
     {
-        $this->title = $this->module->trans('My module title', [], 'Modules.Mymodule.Something);
+        $this->title = $this->trans('My module title', [], 'Modules.Mymodule.Something);
```

Comment:
According to my test, this is the right call. Can someone confirm, maybe @Quetzacoalt91 ?

Reply:
I confirm you this is the right call (according to my test too 😅 )

Reply:
When using trans() with a domain, both are valid.  :)

Reply:
@Quetzacoalt91 If I try to use `$this->module->trans()` in a module front controller with PS 1.7.7, I get the following error:
<img width="1119" alt="Capture d’écran 2019-12-30 à 12 06 12" src="https://user-images.githubusercontent.com/3830050/71579448-df051980-2afc-11ea-9a97-d0bfa67783f2.png">